HackerRank Problem/Python/bigger-is-greater.py

```diff
@@ -0,0 +1,38 @@
+
+import math
+import os
+import random
+import re
+import sys
+
+# Complete the biggerIsGreater function below.
+def biggerIsGreater(w):
+    w = [letter for letter in w.split()[0]]
+    sizeW = len(w)
+    for i in range((sizeW-2), -1, -1):
+        for j in range((i+1), sizeW):
+            if(w[i]<w[j]):
+                tmp = w[i]
+                w[i] = w[j]
+                w[j] = tmp
+                return ''.join(w)
+
+        for j in range(i, sizeW-1):
+            if(w[j]>w[j+1]):
+                tmp = w[j]
+                w[j] = w[j+1]
+                w[j+1] = tmp
+    return "no answer"
+if __name__ == '__main__':
+    fptr = open('input.txt')
```

Comment:
@MediMESS you should provide the input.txt file else please consider giving os environment as output path and open that in write mode.